Mathlib/NumberTheory/NumberField/CanonicalEmbedding/Basic.lean

```diff
@@ -652,6 +659,14 @@ theorem latticeBasis_repr_apply (x : K) (i : ChooseBasisIndex ℤ (𝓞 K)) :
 
 variable (I : (FractionalIdeal (𝓞 K)⁰ K)ˣ)
 
+/-- The image of the fractional ideal `I` in the mixed space. -/
+abbrev idealLattice : Submodule ℤ (mixedSpace K) := LinearMap.range <|
+  (mixedEmbedding K).toIntAlgHom.toLinearMap ∘ₗ ((I : Submodule (𝓞 K) K).subtype.restrictScalars ℤ)
+
+theorem mem_idealLattice (x : mixedSpace K) :
```

Comment:
Maybe `x` implicit? (It seems the standard for iff results, but it is not very important).

Reply:
Actually, I am a big fan of making variable implicit in (most) `iff` statements so I changed it here and also at other places in the file.